### ISSUE TITLE
Newly-created pools not automatically shown for remote lake

### ISSUE THREAD

**Issue by philrz** [brimdata/brim#2545]
Repro is with Brim commit e17764c and Zed commit `b242ea7`.

As shown in the attached video, a newly created pool is automatically reflected if made in the local lake, but if made in a remote lake, I needed to manually click **View > Reload** in order to see it.

https://user-images.githubusercontent.com/5934157/191372005-c382afe9-f319-4220-9f2a-405eba5213a8.mp4

**Comment by philrz** [comment]
I think I see what's going on: All attempts by the client to use the `/events` endpoint are failing due to lack of auth.

The test steps are captured in the attached video. The lower-right shell has evidence that the cloud will see my client's public IP as `199.83.220.115`. The lower-left shell is logged into the cloud host and has a `tcpdump` command line ready to capture my cleartext traffic headed for the cloud service, though I intentionally hold off on starting the capture until after I've entered my credentials with auth0 and I'm about to send control back to my Zui client.

https://user-images.githubusercontent.com/5934157/191394330-c9292453-6d76-4380-be2c-f940b9b5d4ad.mp4

We see that just over 1-second into the capture when the client is connecting, an attempt is made to `GET /events` and the response is `HTTP/1.1 401 Unauthorized`. You can also see from the `X-Forwarded-For: 199.83.220.115` that this is definitely coming from my laptop's public IP.

![image](https://user-images.githubusercontent.com/5934157/191394706-20dac4a5-623a-45ea-a67d-e8406dcad6e2.png)

In conclusion, I assume something more sophisticated needs to be done to make sure the connection to the events endpoint has the benefit of auth similar to that used on the main connection.
